wandb/sdk/artifacts/_models/storage.py

```
@@ -0,0 +1,31 @@
+from __future__ import annotations
+
+from typing import Any, Optional
+
+from pydantic import ConfigDict, StrictStr
+from typing_extensions import Self
+
+from wandb._pydantic import to_camel
+from wandb.sdk.artifacts.storage_layout import StorageLayout
+
+from .base_model import ArtifactsBase
+
+
+class StoragePolicyConfig(ArtifactsBase):
+    model_config = ConfigDict(
```

So this means we will do the string strip and empty check like we were doing explictly?
And if user pass some unknown key in `kwargs`, the constructor would throw error? Or it will silently accept new key?

---

Right now it'll ignore unknown keys (this is default behavior), without accepting them as new attributes.

QQ just in case -- do we want this to be restrictive enough that we raise on unknown keys?  Or alternately -- is there an ask/use case that requires supporting arbitrary keys?

(The former is configured by `extra="forbid"`, the latter by `extra="allow"`)